HHS/Head-Start-TTADP
# [Prod] Assorted new filters, design feedback for training report, devops improvements

## Description of change

- Made many small changes to the training report and session form based on feedback from Lauren and Kelly
- Added "report text" and "groups" to the goal dashboard
- Fix a bug with the TTA history
- Fix a bug around changing the status of objectives on approved ARs
- Added "Specialist name" filter to AR landing and Regional Dashbaord

## How to test

- [Ensure the issues identified here were addressed](https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1841)
- Confirm the new filters on the goal dashboard work as expected (note that report text is very slow)
- Ensure that the TTA history topics graph shows the correct number of ARs
- For 1834, the LDM model test updates, it is considered tested if the CI passes
- For users with the ability to approve reports should see the "Specialist name" filter.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1841
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1571
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1834
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1823
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1848
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-448


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
